app/views/artefacts/form/_tags.html.erb

```diff
@@ -1,56 +1,5 @@
 <%- unless f.object.app_without_tagging_support? %>
   <%- if f.object.tagging_migrated? %>
```

Comment:
Yeah. Perhaps you can replace the entire contents of this template with the message from `migrated_app.html.erb`.